src/Components/Aspire.Pomelo.EntityFrameworkCore.MySql/AspireEFMySqlExtensions.cs

```diff
@@ -39,12 +35,9 @@ public static partial class AspireEFMySqlExtensions
     /// </para>
     /// </remarks>
     /// <exception cref="ArgumentNullException">Thrown if mandatory <paramref name="builder"/> is null.</exception>
-    /// <exception cref="InvalidOperationException">Thrown when mandatory <see cref="PomeloEntityFrameworkCoreMySqlSettings.ConnectionString"/> is not provided.</exception>
-    public static void AddMySqlDbContext<[DynamicallyAccessedMembers(RequiredByEF)] TContext>(
+    public static void AddMySqlEntityFrameworkCore<[DynamicallyAccessedMembers(RequiredByEF)] TContext>(
```

Comment:
Using the `Add` prefix on these methods is feeling a bit clumsy. In the simplest use case, the user will have something like the following right?

```csharp
var connectionString = builder.Configuration.GetConnectionString("mysqldb")
    ?? throw new InvalidOperationException("Connection string is not configured.");
builder.AddMySqlEntityFrameworkCore<AppDbContext>();
builder.Services.AddDbContextPool<AppDbContext>(options => options.UseMySql(connectionString));
```

Could we consider chaining off of `IServiceCollection` in this case instead, e.g.:

```csharp
var connectionString = builder.Configuration.GetConnectionString("mysqldb")
    ?? throw new InvalidOperationException("Connection string is not configured.");
builder.Services.AddDbContextPool<AppDbContext>(options => options.UseMySql(connectionString))
    .EnrichMySqlDbContext<AppDbContext>(builder.Configuration);
```

Naming is hard for sure for this, but some suggestions for the prefix:

- `Enrich` (possibly conflicts with log/trace enrichment)
- `Instrument` (is it complete enough? are health checks and resilience part of instrumenting?)
- `Aspirify` (😄)
- `Configure` (too overloaded)
- ?

Reply:
Some of this methods will also need the `IHostApplicationBuilder`, so it will look like this instead:

```c#
builder.Services.AddDbContextPool<AppDbContext>(options => options.UseMySql(connectionString))
    .EnrichMySqlDbContext<AppDbContext>(builder);
```

Would it help to keep the two extension methods? One on `IServiceCollection` and the one on `IHostApplicationBuilder`?

Reply:
Applied your suggestions.